modules/tide_media/tide_media.module

```diff
@@ -464,3 +466,35 @@ function tide_media_file_presave(FileInterface $file) {
     $file->setFilename($filename_new);
   }
 }
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function tide_media_preprocess_page(&$variables) {
+  $route_name = \Drupal::routeMatch()->getRouteName();
+  $current_user = \Drupal::currentUser();
+  $roles = $current_user->getRoles();
+
+  if ($route_name === 'entity.user.edit_form' && in_array('secure_file_user', $roles)) {
+    $url = Url::fromRoute('entity.user.canonical', ['user' => $current_user->id()]);
+    $response = new RedirectResponse($url->toString());
+    $response->send();
+    exit();
```

Comment:
Please don't use exit. It is not suggested to use it in drupal.
Why cant we make the access False for this form ?